packages/next/client/image.tsx

```diff
@@ -698,15 +698,21 @@ export default function Image({
       }
     }
   }
-
   const imgStyle = Object.assign({}, style, layout === 'raw' ? {} : layoutStyle)
+  const svgBlurPlaceholder = `url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http%3A//www.w3.org/2000/svg' xmlns%3Axlink='http%3A//www.w3.org/1999/xlink' viewBox='0 0 ${widthInt} ${heightInt}'%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='50'%3E%3C/feGaussianBlur%3E%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'%3E%3C/feFuncA%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Cimage filter='url(%23b)' x='0' y='0' height='100%25' width='100%25' xlink%3Ahref='${blurDataURL}'%3E%3C/image%3E%3C/svg%3E");`
   const blurStyle =
     placeholder === 'blur' && !blurComplete
       ? {
-          filter: 'blur(20px)',
           backgroundSize: objectFit || 'cover',
-          backgroundImage: `url("${blurDataURL}")`,
           backgroundPosition: objectPosition || '0% 0%',
+          ...(layout === 'raw' && blurDataURL?.startsWith('data:image')
```

Comment:
Whats the significance of `blurDataURL?.startsWith('data:image')`? Does xlink not work when embedding a URL to an external image?

Reply:
Yeah, I'm actually not 100% on why, but the blurry placeholder never displays when the  SVG xlink (or now the href) is an external URL. I can look into this more, but for now it seems ok to leave this in for dev mode?

Reply:
Yeah I think thats fine for now. The tests will just have to check for different html in dev vs prod mode